shell/client/styles/_topbar.scss

```diff
@@ -741,6 +741,11 @@ body>.topbar {
           left: (($navbar-width-desktop-shrunk - 24px) / 2);
           width: 24px;
           height: 24px;
+          @media #{$mobile} {
+            left: (($navbar-width-desktop-shrunk - 40px) / 2);
```

Comment:
I don't quite understand the intended high-level meaning of this constant.

Reply:
TBH your guess is as good as mine at this point...